docs/universal-gateway/examples/offload-analytics.mdx

```diff
@@ -0,0 +1,93 @@
+
+ngrok and Traffic Policy let you integrate your ["front door" gateway](/docs/universal-gateway/examples/front-door-pattern) with an analytics service you host within your network to capture vital information about your traffic as a part of the request-response lifecycle.
+
+With this gateway setup and the http-request action, you can:
+
+- Host your analytics service in any network, region, or cloud.
+- Leave your analytics service completely unexposed to the public internet.
+- Simplify your upstream services by removing libraries and code for instrumenting them directly.
+- Configure for exactly the timeout and retry conditions your services require.
+
+## 1. Create endpoints for your services
+
+Start an internal [Agent Endpoint](/docs/universal-gateway/agent-endpoints/), replacing `$PORT` based on where your upstream service listens.
+You can also use one of our [SDKs](/docs/agent-sdks) or the [Kubernetes Operator](/docs/k8s).
+
+```bash
+ngrok http $PORT --url https://service.internal
+```
+
+Start a second endpoint for your analytics service.
+
+```bash
+ngrok http $PORT --url https://analytics.internal
+```
+
+## 2. Reserve a domain
+
+<ReserveDomain />
+
+## 3. Create a Cloud Endpoint
+
+<CloudEndpoint />
+
+## 4. Request customer data from your CRM and apply rate limiting with Traffic Policy
+
+While still viewing your new cloud endpoint in the dashboard, copy and paste the policy below into the Traffic Policy editor.
+
+```yaml
+on_http_request:
+  - actions:
+      - type: forward-internal
+        config:
+          url: https://service.internal
+
+on_http_response:
+  - actions:
+      # Send authenticated request to analytics service
+      - type: http-request
+        config:
+          url: https://analytics.internal/api/request
+          method: POST
+          headers:
+            Authorization: "Bearer <ANALYTICS_API_TOKEN>"
+            Content-Type: "application/json"
+          body: |
+            {
+              "path": "${req.url.path}",
+              "method": "${req.method}",
+              "status_code": "${res.status_code}",
+              "country_code": "${conn.geo.country_code}",
+              "region": "${conn.server_region}",
+            }
+          timeout: 1s
+```
+
+### What's happening here?
+
+This policy sends all requests directly to your upstream service at `https://service.internal`.
+
+As part of the response lifecycle, this policy also sends an authenticated request to your analytics service with a body containing attributes, injected with CEL interpolation, about the user's request and your server's response.
+The `http-request` action is purposely given a short timeout and no retry logic to ensure this analytics request doesn't add unnecessarily slow your service's response time.
+
+## 5. Try out your endpoint
+
+<TryOut />
+
+## What's next?
+
+- Read more about [Traffic Policy](/docs/traffic-policy) and its [core concepts](/docs/traffic-policy/concepts).
```

Comment:
```suggestion
- Explore our other examples of using the [`http-request`](/docs/traffic-policy/actions/http-request/#examples) and [`forward-internal`](/docs/traffic-policy/actions/forward-internal/#examples) Traffic Policy actions.
```

Reply:
@joelhans I think it'd be good to go through the whole PR and update the Traffic Policy bullet in the "What's next" sections to be in this format, linking to examples of the actions used on the page